src/field/scrcmd_pokemon_misc.c

```diff
@@ -450,10 +450,10 @@ BOOL ScrCmd_CommSanitizeParty(ScriptContext *ctx) {
 }
 
 BOOL ScrCmd_SetMonForm(ScriptContext *ctx) {
-    FieldSystem *fsys = ctx->fsys;
+    FieldSystem *fieldSystem = ctx->fieldSystem;
     u16 index = ScriptGetVar(ctx);
     u16 form = ScriptGetVar(ctx);
-    Pokemon *mon = GetPartyMonByIndex(SaveArray_PlayerParty_Get(fsys->savedata), index);
+    Pokemon *mon = GetPartyMonByIndex(SaveArray_Party_Get(fieldSystem->saveData), index);
```

Comment:
SaveArray_GetParty? Save_GetParty?

Reply:
According to the Module_Behaviour style guidelines for functions, this should be `Party_GetPlayerPartyFromSaveArray` or `Party_GetSaveArrayPlayerParty`. But I'm not even sure if `SaveArray` is a good name (looks closer to a save section type thing)